packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -951,6 +951,20 @@ const defaultConfigs = {
         invertPrice: true
       }
     }
+  },
+  USDDEXTF: {
+    type: "expression",
+    expression: "ETHDEXTF * USDETH",
+    lookback: 7200,
+    minTimeBetweenUpdates: 60,
+    twapLength: 3600,
```

Comment:
The UMIP specifies a 5 minute TWAP, so to meet UMIP specs `twapLength` should be `300`

Reply:
File updated to use 300 secs